### ISSUE TITLE
add Wikipedia to citation formats

### ISSUE THREAD

**Issue by ebenenglish** [boston-library/commonwealth-vlr-engine#134]
When I am on the item detail page (catalog#show)
And I click the "Cite" link in the Tools widget
I should see a section with the Wikipedia citation format

See https://digitalcollections.nypl.org/items/ed807500-7058-013c-bffa-0242ac110005 for an example.

**Comment by jwillbpl** [comment]
I started implementing last1, first1, last2, first2, etc. parameters, until I came across institution names and inconsistent authors entries (like "Trustman, Benjamin A., Trustman, Julia M." in one array item instead of two).
The "vauthors" parameter is also difficult to implement, because of the same inconsistencies.
For now, I omitted mentioning authors; the author fields are only suggested and not required.

NYPL uses "author=Digital Collections, The New York Public Library" on all their items.

Useful links regarding citation templates:
https://en.wikipedia.org/wiki/Wikipedia:Citation_templates
https://en.wikipedia.org/wiki/Template:Cite_web

Let me know what you think.
